ivy_tests/test_ivy/test_functional/test_core/test_gradients.py

```diff
@@ -239,9 +239,7 @@ def func(xs):
 @pytest.mark.parametrize("nth", [1, 2, 3])
 def test_grad(x, dtype, func, backend_fw, nth):
     # ToDo: Remove skipping for paddle and jax for nth > 1
-    if backend_fw == "numpy" or (
-        (backend_fw == "paddle" or backend_fw == "jax") and nth > 1
-    ):
+    if backend_fw == "numpy" or backend_fw in ["paddle", "jax"] and nth > 1:
```

Comment:
I think this would be clearer if we left the brackets
```suggestion
    if backend_fw == "numpy" or (backend_fw in ["paddle", "jax"] and nth > 1):
```

Reply:
Hey @KareemMAX, I made the change please look into it now, thank you.